packages/contracts-bedrock/test/L1/OptimismPortal.t.sol

```diff
@@ -364,7 +541,7 @@ contract OptimismPortal_FinalizeWithdrawal_Test is CommonTest {
             target: bob,
             value: 100,
             gasLimit: 100_000,
-            data: hex""
+            data: hex"aa"
```

Comment:
Why was this changed?

Reply:
Ah my guess its because the call in the custom gas token withdrawal case makes a call since its conditional based on the length, can see [ab69944](https://github.com/ethereum-optimism/optimism/pull/10143/commits/ab6994432a5485ab8fea0197fd0077b9b4d4fde8)

Reply:
I think we should revert this back to having calldata. I wish that "proof verify" was an external call so we could just mock a `return true` without needing to have a real proof, that would make all of these tests way easier

Reply:
I think we should add back the calldata, its a noop in call cases except for the custom gas token withdrawal which lets us test the outbound call case

Reply:
Yes this is the case:

```
Failing tests:
Encountered 1 failing test in test/L1/OptimismPortal.t.sol:OptimismPortalWithMockERC20_Test
[FAIL. Reason: expected call to 0x1D96F2f6BeF1202E4Ce1Ff6Dad0c2CB002861d3e with data 0x, value 0 to be called 1 time, but was called 0 times] test_finalize
WithdrawalTransaction_provenWithdrawalHash_nonEther_succeeds() (gas: 414746)
```

I would like to leave in the `aa` for now

Reply:
Thanks for looking into this, resolving the thread